README.md

```diff
@@ -1,61 +0,0 @@
-# CIROH Cloud Terraform Configuration
```

Comment:
Seems like we need to preserve some of this as a `Terraform_README.md`

Reply:
Yeah these were just Matt D.'s notes on the prototype from a while back